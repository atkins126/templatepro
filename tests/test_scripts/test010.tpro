{{:obj.Prop1}}
YY
{{:value0}}ABC{{:value1}}DEF{{:value2}}

{{:value3}}


{{:value4}}-GHI-{{:value5}}-JKL-{{:value6}}

{{:obj.Prop1}}
XXX
{{:obj.Prop2}}
YYY
