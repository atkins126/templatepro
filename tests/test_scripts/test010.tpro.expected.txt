value1.1
YY
trueABCtrueDEFDANIELE2

DANIELE3


DANIELE4-GHI-DANIELE5-JKL-DANIELE6

value1.1
XXX
value2.1
YYY
